app/lib/services.rb

```diff
@@ -13,9 +13,14 @@ def self.content_store
   end
 
   def self.cached_content_item(base_path)
-    Rails.cache.fetch("finder-frontend_content_items#{base_path}", expires_in: 5.minutes) do
+    content_item = content_store.content_item(base_path)
+    max_age = content_item.cache_control["max-age"]
+
+    Rails.cache.fetch("finder-frontend_content_items#{base_path}", expires_in: max_age) do
```

Comment:
You can keep `max_age` as 5 minutes here - we don't mind how long we cache content items, it's just how long we cache the whole page that matters.